README.md

```diff
@@ -1,57 +1,143 @@
 
 ## Setup Local Environment for Development
 
-### Environment variables
+### Prerequisites
 
-`cp .env.example .env`
+Before you start, ensure you have the following installed:
 
-Change the values for your local development (e.g. for running a local PostgreSQL database).
+- [Node.js](https://nodejs.org/) (version 14 or higher)
+- [npm](https://www.npmjs.com/) (version 6 or higher)
+- [PostgreSQL](https://www.postgresql.org/)
+- [Prisma](https://www.prisma.io/) CLI
 
-### Run
+### 1\. Clone the Repository
 
-`npm run dev`
+```
+git clone https://github.com/Glo-Foundation/glo-wallet.git
+cd glo-wallet
+```
 
-## Env variables
+### 2\. Install Dependencies
 
-Env variables are stored in [Vercel](https://vercel.com/glodollar/glo-wallet/settings/environment-variables).
+```
+npm install
+```
 
-The easiest way to retrieve is to use [Vercel CLI](https://vercel.com/docs/cli) and run `vercel env pull`. Make sure you link your repo first with `vercel link`.
+### 3\. Set Up Environment Variables
+
+Copy the example environment variables file and modify it for your local setup:
+
+```
+cp .env.example .env
+```
+
+Update the `.env` file with your local development values (e.g., for running a local PostgreSQL database).
+
+### 4\. Set Up PostgreSQL Database
+
+If you don't have PostgreSQL installed, you can install it using Homebrew on macOS:
+
+```
+brew install postgresql
+```
+
+For Linux users, you can install PostgreSQL using the following commands:
+
+```
+sudo apt update
+sudo apt install postgresql postgresql-contrib
+```
+
+For Windows users, download and install PostgreSQL from the [official website](https://www.postgresql.org/download/windows/).
+
+Start the PostgreSQL service:
+
+```
+brew services start postgresql
+```
+
+For Linux users:
+
+```
+sudo service postgresql start
+```
+
+For Windows users, start the PostgreSQL service from the Services application or using the pgAdmin tool.
+
+Create a new PostgreSQL database:
+
+```
+createdb mydatabase
+```
+
+Create a new PostgreSQL user with a password (optional):
+
+```
+psql postgres
+CREATE USER myuser WITH PASSWORD 'mypassword';
+ALTER USER myuser WITH SUPERUSER;
+```
+
+Update the `.env` file with the PostgreSQL connection string:
+
+```
+POSTGRES_URL=postgresql://myuser:mypassword@localhost:5432/mydatabase
+```
+
+### 5\. Run Database Migrations
+
+```
+npx prisma migrate deploy
+```
+
+### 6\. Start the Development Server
+
+```
+npm run dev
+```
+
+### 7\. Access the UI
+
+Open your web browser and navigate to `http://localhost:3000` to access the Glo Wallet UI.
+
+## Environment Variables
+
+Environment variables are stored in [Vercel](https://vercel.com/glodollar/glo-wallet/settings/environment-variables). The easiest way to retrieve them is to use the [Vercel CLI](https://vercel.com/docs/cli) and run `vercel env pull`. Make sure you link your repo first with `vercel link`.
```

Comment:
Most people won't have access to this. Let's replace with instructions for which environment examples to use / which you need to replace to setup.

Did you actually run the project in its entirety yourself in writing this guide? Can you play around with it and make sure everything works?

Reply:
You mean something like this?

## Environment Variables

Below are examples of the environment variables required for the setup. Replace the placeholder values with your own:

```

NEXT_PUBLIC_GOERLI_RPC_URL="https://ethereum-goerli.publicnode.com"
NEXT_PUBLIC_ALFAJORES_RPC_URL="https://alfajores-forno.celo-testnet.org"
NEXT_PUBLIC_OPTIMISM_SEPOLIA_RPC_URL="https://sepolia.optimism.io"
NEXT_PUBLIC_ARBITRUM_SEPOLIA_RPC_URL="https://sepolia-rollup.arbitrum.io/rpc"
NEXT_PUBLIC_BASE_SEPOLIA_RPC_URL="https://base-sepolia-rpc.publicnode.com"
NEXT_PUBLIC_VECHAIN_RPC_URL="https://rpc.vechain.energy"

WEBHOOK_API_KEY="your-webhook-api-key"
CMC_API_KEY="your-cmc-api-key"
NEXT_PUBLIC_ALCHEMY_KEY="your-alchemy-key"
MORALIS_API_KEY="your-moralis-api-key"
KV_URL="your-kv-url"
KV_REST_API_URL="your-kv-rest-api-url"
KV_REST_API_TOKEN="your-kv-rest-api-token"
KV_REST_API_READ_ONLY_TOKEN="your-kv-rest-api-read-only-token"

VERCEL_URL="localhost:3000"
POSTGRES_URL="postgresql://myuser:mypassword@localhost:5432/mydatabase"
```

And yeah, I ran the project
<img width="1440" alt="Screenshot 2024-12-10 at 23 17 47" src="https://github.com/user-attachments/assets/bd28ef27-04de-4a49-b42f-01b52cb6d900">

Reply:
Yeah something like that exactly!

Aside from this first screen, were you able to completely use the project as well?